pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -421,7 +421,9 @@ func (p *testPlanner) changeVersionSteps(
 	var steps []testStep
 	for j, node := range previousVersionNodes {
 		steps = append(steps, p.newSingleStep(
-			restartWithNewBinaryStep{version: to, node: node, rt: p.rt, settings: p.clusterSettings()},
+			restartWithNewBinaryStep{
+				version: to, node: node, rt: p.rt,
```

Comment:
@DarrylWong seems like we're no longer passing the cluster settings when restarting binaries here. I think this is a bug?

Reply:
Ah yes thank you, that's definitely not correct. Will need to test it out but might be why [change-replicas/mixed-version](https://github.com/cockroachdb/cockroach/issues/127106) is failing. Still chasing another bug related to the datadriven test thats blocking the backports (I think I just need to bump the version). I'll put out a fix for them.

Reply:
Yes, we're not passing the `install.EnvOption{"COCKROACH_SCAN_MAX_IDLE_TIME=10ms"}` anymore which is likely why the test started to fail.